cardano-api/src/Cardano/Api.hs

```diff
@@ -160,6 +160,9 @@ module Cardano.Api (
     createAndValidateTransactionBody,
     makeTransactionBody, -- TODO: Remove
     TxBodyContent(..),
+    defaultTxBodyContent,
+    defaultTxFee,
+    defaultTxValidityUpperBound,
     setTxIns,
```

Comment:
We can potentially add a little heading e.g `-- ** Transaction body builders` or something like that

Reply:
Done!